### PR TITLE
updated in TaskDevelopment

### PR DESCRIPTION
**Pull request by qm-sasmin**
made changes in the programs.

### CODE REVIEW

File: NumberReverse/Program.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ static void Main(string[] args)
             // Asking the user to enter a number
             Console.WriteLine("Enter your num");
             int number = Convert.ToInt32(Console.ReadLine());
-            string emptyString = "";
+            string empty = "";
```
Comment (qm-bhupal) [inline NumberReverse/Program.cs]: use int and give a proper name like reverseNumber

Comment (qm-sasmin) [reply NumberReverse/Program.cs]: Ok

File: NumberReverse/Program.cs [hunk 2]
```diff
@@ -10,7 +10,8 @@ static void Main(string[] args)
             // Asking the user to enter a number
             Console.WriteLine("Enter your num");
             int number = Convert.ToInt32(Console.ReadLine());
-            string empty = "";
+            string reverse = "";
+            int reverseNumber;
```
Comment (qm-bhupal) [inline NumberReverse/Program.cs]: write the logic using int, don't use string

**Review by qm-bhupal — CHANGES_REQUESTED** [review]

File: NumberReverse/Program.cs (hunk 2 above)
Comment (qm-sasmin) [reply NumberReverse/Program.cs]: Ok

**Pull request merged**